semgrep-core/src/matching/Generic_vs_generic.ml

```diff
@@ -3003,16 +3003,15 @@ and m_directive a b =
 
   m_directive_basic a.d b.d >!> fun () ->
   match a.d with
-  (* normalize only if very simple import pattern (no alias) *)
-  | G.ImportFrom (_, _, _, None)
-  | G.ImportAs (_, _, None) -> (
-      (* equivalence: *)
-      let normal_a = Normalize_generic.normalize_import_opt true a.d in
-      let normal_b = Normalize_generic.normalize_import_opt false b.d in
-      match (normal_a, normal_b) with
-      | Some (a0, a1), Some (b0, b1) ->
-          m_tok a0 b0 >>= fun () -> m_module_name_prefix a1 b1
-      | _ -> fail ())
+  (* normalize only if very simple import pattern (no aliases) *)
+  | G.ImportFrom (_, _, imports)
+    when List.for_all
+           (function
+             | _, None -> true
```

Comment:
nit: this would be more readable for me if there were underscored names given to the wildcards. In particular, something like `(_name, Some(_aliases))` would let me know what we're filtering for. It's not the norm in this file though so this is fine

Reply:
I've made this change and added a comment. Hopefully that's clearer now.